Backend-API/src/main/java/com/avangers/backendapi/services/UserService.java

```diff
@@ -0,0 +1,8 @@
+package com.avangers.backendapi.services;
+
+import com.avangers.backendapi.models.User;
+
+public interface UserService {
+
+    void addUser(User user);
```

Comment:
This should throw an error or return with something, we should let the user know if the registration was successfull.